torch/distributed/algorithms/quantization/quantization.py

```diff
@@ -116,6 +116,8 @@ def auto_quantize(func, qtype, quant_loss=None):
     """
     @functools.wraps(func)
     def wrapper(*args, **kwargs):
+        out_splits = kwargs.get('out_splits', None)
```

Comment:
what's these two variables used for? shouldn't we place these to the section of `all_to_all_single` as it's only specific to `all_to_all_single`?